client-frameworks-support/client-support-angular/package.json

```diff
@@ -8,7 +8,9 @@
     "test": "ng test",
     "lint": "ng lint",
     "build:lib": "ng build client-support-angular --prod",
-    "release": "npm run build:lib && cd dist/client-support-angular/ && npm publish"
+    "release": "npm run build:lib && cd dist/client-support-angular/ && npm publish",
```

Comment:
I would not keep release logic in this file, do something like this: https://github.com/SAP/luigi/blob/master/scripts/npmPublish.sh#L130 : same as these lines you pass the client-support-angular folder and it will publish according to the package.json found inside that folder

Reply:
Removed